models.yml

```
@@ -765,6 +765,14 @@ committee:
     type: relation-list
     to: committee/forward_to_committee_ids
     restriction_mode: B
+  forward_agenda_to_committee_ids:
```

If this is an allow list, maybe it would be possible to represent this in the name. Adopting the schema of the back relation would result to 
```suggestion
  send_agenda_forwardings_to_committee_ids:
```
but describe more of a fixed recipient list.